File: uri/media/media.kt
import java.util.Scanner
import kotlin.text.*

fun media( notaA : Double, notaB : Double ) : Double {

    return ( ((notaA * 3.5) + (notaB * 7.5) ) / 11.0 )
}

fun main(args: Array<String>){
    var input = Scanner(System.`in`)
    var notaA : Double;
    notaA = (input.nextLine()).toDouble();

    var notaB : Double;
    notaB = (input.nextLine()).toDouble();

    var resposta : Double    
    resposta = media(notaA, notaB)
    println("MEDIA = %.5f".format(resposta))
}

File: uri/media/media.py
def media( notaA, notaB ):
    return ((notaA * 3.5) + (notaB * 7.5)) / 11.0

notaA = float(input())
notaB = float(input())
med = media(notaA, notaB)
print "MEDIA = {0:.5f}".format(med)
# print ("MEDIA = {0:.5f}".format(med))